examples/search_bar/main.rs

```diff
@@ -0,0 +1,33 @@
+use gtk::prelude::*;
+
+fn main() {
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.search_bar"),
+        Default::default(),
+    );
+    application.connect_activate(build_ui);
+    application.run();
+}
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(application);
+    window.set_title(Some("Search Bar"));
+
+    let search_bar = gtk::SearchBar::new();
+    search_bar.set_valign(gtk::Align::Start);
+    window.set_child(Some(&search_bar));
+    search_bar.set_key_capture_widget(Some(&window));
+
+    let search_box = gtk::Box::new(gtk::Orientation::Horizontal, 6);
```

Comment:
I would use a Vertical container here, put the `SearchBar` on it and a label, and update the label with whatever the user types on the search entry

Reply:
Done